Misc/NEWS.d/next/Core and Builtins/2021-05-04-21-55-49.bpo-44024.M9m8Qd.rst

```diff
@@ -0,0 +1,3 @@
+Use the common :exc:`TypeError` message for non-string second arguments
```

Comment:
```suggestion
Improve the error message for non-string second arguments
```
I don't think "the common message" will make much sense to most people reading this.

Reply:
You are right. Updated.